src/standard-schema.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Standard Schema types and utilities
+ * This allows the library to work with any Standard Schema compliant validation library
+ * (Zod, Valibot, ArkType, etc.)
+ */
+
+import type { StandardSchemaV1 } from '@standard-schema/spec';
+
+// Import official Standard Schema types
+export type { StandardSchemaV1 } from '@standard-schema/spec';
```

Comment:
```suggestion
```
We should import `StandardSchemaV1` from the schema package directly when using it in other places